OrbitClientData/TracepointEventBuffer.cpp

```diff
@@ -89,7 +89,12 @@ uint32_t TracepointEventBuffer::GetNumTracepointsForThreadId(int32_t thread_id)
       return num_total_tracepoints_;
     }
     if (thread_id == -1 /*TODO(b/166238019) Use a proper constant again*/) {
```

Comment:
Since the constant is no longer used, can there at least be a comment on what this -1 represents, at least in the TODO?

Reply:
I changed the comment.